cola/qtutils.py

```diff
@@ -440,7 +440,7 @@ def open_files(title, directory=None, filters=''):
     return result[0]
 
 
-def opendir_dialog(caption, path):
+def opendir_dialog(path, caption='Browse...'):
```

Comment:
I think this would have to updated at the call site since different call sites have different captions. I guess `Browse...` would be okay. I figured the more specific caption was helpful.

Would an alternative for that issue be to use a more specific caption such as, "Open Git Repository..."?

Alternatively, if we wanted to make most of the call sites not specify a value, then it'd be best to use the value of `None` as the default value here, and then when it's `None` we can call `N_('Browse...')` to get the caption within the function body.

We have to defer the call to `N_(...)` until after the translations have been activated so we cannot `N_(...)` when providing default values for keyword arguments. Keyword arg defaults are global variables evaluated at module scope (not deferred) which is why we'd have to use the `None` trick to defer it.